index.html

```diff
@@ -12,7 +12,7 @@
 <body>
   <header class="menu-wrap">
     <nav class="nav-bar">
-      <a href="#" class="brand">My Logo</a>
+      <a href="#" class="brand">Jesse</a>
```

Comment:
good job so far @Jessekay , just make this logo color a bit darker as it's contrast ratio is not enough you can use the tools mentioned in the lesson to check if the contrast is good